ls1intum/Artemis
# `Modeling exercises`: Improve grid appearance for Apollon models

### Checklist
#### General



#### Client

### Motivation and Context
This PR simply changes the grid color of Apollon in Artemis. 
With the changed color, the bigger grid looks more distinguished than the smaller ones.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Log in to Artemis
2. Navigate to Course Administration
3. Create a Modeling exercise for any one of the exercises
4. Observe that the bigger grid of the canvas is distinguished from smaller grids (as illustrated in the screenshot section below).

#### Code Review

### Screenshots
#### Before
![image](https://user-images.githubusercontent.com/14681902/191361431-37547345-7c59-463b-a016-4358515795a4.png)
#### After
![image](https://user-images.githubusercontent.com/14681902/191362338-fa05b9c3-4341-4710-aa3f-e96866cde7e2.png)
